tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -153,8 +160,7 @@ def generate_metadata(list, args):
     generate_version(metadata_list, args.package_version)
     generate_authors(metadata_list, 'Microsoft')
     generate_owners(metadata_list, 'Microsoft')
-    generate_description(metadata_list, 'This package contains native shared library artifacts '
-                                        'for all supported platforms of ONNX Runtime.')
+    generate_description(metadata_list, args.package_name)
```

Comment:
The description for the WinML nuget accidently used the same description as that of OnnxRuntime's nugets

Reply:
Is this necessary for the fix?

Reply:
Based on offline sync, kept this minor change